ouroboros-consensus-shelley/ouroboros-consensus-shelley.cabal

```diff
@@ -73,6 +75,7 @@ library
 
   default-language:    Haskell2010
   ghc-options:         -Wall
+                       -Werror
```

Comment:
Please remove this; we develop locally without it (you can opt-in via a `cabal.project.local` file) while CI does enable it. (EG search for it in `nix/haskell.nix` if you're curious.)